src/Controller/ArticleController.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Article;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\Request;
+
+class ArticleController extends AbstractController
+{
+//    /**
+//     * @Route("/articles", name="article")
+//     */
+//    public function index(): Response
+//    {
+//        return $this->render('article/index.html.twig', [
+//            'controller_name' => 'ArticleController',
+//        ]);
+//    }
+
+    /**
+     * @Route("/articles/create", name="create_article")
+     */
+    public function createArticle(): Response
```

Comment:
This does not seem to be an operational endpoint? I.e. uses hard-coded values. Maybe it can also be removed.

Reply:
Sorry forgot to remove this. This was from the official symfony getting started tutorial.